pallets/did/src/mock.rs

```diff
@@ -92,8 +92,9 @@ parameter_types! {
 	pub const MaxUrlLength: u32 = 200u32;
 	#[derive(Debug, Clone, PartialEq)]
 	pub const MaxTotalKeyAgreementKeys: u32 = 10u32;
+	// IMPORTANT: Needs to be at least MaxTotalKeyAgreementKeys + 3 (auth, delegation, attestation keys) for benchmarks!
 	#[derive(Debug, Clone)]
-	pub const MaxPublicKeysPerDid: u32 = 10u32;
+	pub const MaxPublicKeysPerDid: u32 = 13u32;
```

Comment:
```suggestion
	pub const MaxPublicKeysPerDid: u32 = MaxTotalKeyAgreementKeys + 3;
```

Reply:
Clippy complains about this except if I implement `Add` for `MaxTotalKeyAgreementKeys`. Also tried to add by using saturated into but that would also have to be implemented. Thus, I would prefer keeping it at 13 then except if you have a better solution.

Reply:
Oh yeah, true. Nevermind then 👍